src/Cdn77/Sniffs/Classes/SingleLineMethodDeclarationWhenPossibleSniff.php

```diff
@@ -51,7 +51,9 @@ public function process(File $file, $pointer) : void
         $tokens = $file->getTokens();
 
         $lineStartPtr = $file->findFirstOnLine(T_WHITESPACE, $pointer);
-        assert(is_int($lineStartPtr));
+        if ($lineStartPtr === false) {
```

Comment:
What's ptr?

Reply:
pointer, standard in cs sniffs

Reply:
Can we rather make it "pointer" so we drop some WTFs from code? The +4 symbol cost is not significant :)

Reply:
done